bee-message/src/payload/milestone/mod.rs

```diff
@@ -6,19 +6,23 @@
 mod essence;
 mod milestone_id;
 
+///
```

Comment:
Missing Docs :trollface:

Reply:
Later

Reply:
there are quite alot of these if i remember correctly, but guess this is something that will be done with an future PR, right?